lib/serverspec/type/hdfs.rb

```diff
@@ -0,0 +1,16 @@
+require 'nokogiri'
```

Comment:
Nokogori is too heavy to install for the users which do not need use this type.
So please change like this.

```ruby
begin
  require 'nokogiri'
rescue LoadError
  fail "nokogiri is not available. Try installing it."
end
```